resources/views/components/post-item.blade.php

```diff
@@ -34,6 +34,10 @@
     </div>
 
     <div class="mt-2">
+        @if ($post->formatting == 'rich')
+        {!! $post->content !!}
+        @else
         {!! nl2br(e($post->content)) !!}
+        @endif
```

Comment:
Would this work?
```suggestion
        {!! $post->formatting == 'rich' ? $post->content : nl2br(e($post->content)) !!}
```

Reply:
Also curious if the `$post->formatting == 'rich` could be put into a method on the model itself? Or even this entire thing as an accessor?

Reply:
Good call, I have added this as an attribute on the Post model, and simplified the logic with the ternary operator